st2api/st2api/controllers/v1/actionexecutions.py

```diff
@@ -376,7 +376,7 @@ def get_all(self, exclude_attributes=None, **kw):
         if 'timestamp_lt' in kw:
             query_options = {'sort': ['-start_timestamp', 'action.ref']}
             kw['query_options'] = query_options
-        elif 'timestamp_gt' in kw:
+        elif 'timestamp_gt' in kw or 'sort_desc' in kw:
```

Comment:
yes, I think I can wrap my head around that.  I like explicitly including them as well.